code/game/machinery/OpTable.dm

```diff
@@ -48,19 +48,19 @@
 		return FALSE
 
 
-/obj/machinery/optable/MouseDrop_T(atom/movable/O as mob|obj, var/mob/P)
-	if(usr.stat || (!ishuman(P) && !isrobot(P)) || !check_table())
+/obj/machinery/optable/MouseDrop_T(atom/movable/O, mob/user)
+	if(!ishuman(user) && !isrobot(user)) //Only Humanoids and Cyborgs can put things on this table
 		return
-	if(!check_table()) //Is the Operating Table currently occupied?
+	if(!check_table()) //Is the Operating Table empty?
 		return
-	if(P.restrained() || P.buckled || P.incapacitated()) //Folks who are restrained or otherwise incapacitated cannot get on the table
+	if(user.restrained() || user.buckled || user.IsWeakened() || user.stunned || user.incapacitated()) //Is the person trying to use the table incapacitated or restrained?
```

Comment:
this does not check for `user.stat` anymore, and i'm not sure it's covered by incapacitateted() (it might!)
Please double-check whether uncoscious people can drag stuff.

Reply:
Hmm. Other MouseDrop_T() procs don't do the `stat` check, so I removed it. I'll double check!

Reply:
Confirmed, you can **not** mouse-drag while asleep. Removing this check appears to be okay!
I don't think it matters if it is covered by `incapacitated()` as `user.stat` check does not appear in similar `MouseDrop_T()` procs (looking at body scanner for instance) and you still cannot drag while asleep into/onto those objects.
I wonder if BYOND had an update that changed this since OpTable was originally written. Either way, it all seems good!